docs/docs/api/map-matching/api-reference.md

```diff
@@ -175,7 +175,9 @@ Each `edge` may include:
 | Edge item | Description |
 | :--------- | :---------- |
 | `names` | List of names. |
-| `length` | Edge length in the units specified. The default is kilometers. |
+| `source_percent_along` | The start of an edge's match as percentage of its length in (0, 1) range. If an edge was fully matched, we omit this value.
+| `target_percent_along` | The end of an edge's match as percentage of its length in (0, 1) range. If an edge was fully matched, we omit this value.
```

Comment:
we should add unit tests but gah its so trivial

Reply:
my thought exactly